roles/sat6repro/tasks/main.yml

```diff
@@ -91,11 +91,14 @@
 - include: restore.yml
   when: perform_restore
 
-- name: Restart katello-service and test satellite
-  command: "{{ item }}"
-  with_items:
-    - katello-service start
-    - hammer ping
+- name: Restart katello-service
+  command: katello-service start
+
+- name: Wait for foreman-tasks service to comes up
+  shell: sleep 5m
```

Comment:
@sthirugn : I was reading about ansible before making change and realized that pause is for playbook but not for tasks. But as you suggested 'pause' looks more promising. will update the PR.